nova6/nova6.py

```diff
@@ -174,11 +184,11 @@ def parse_args(args=None):
                         help='Outputs an XML showing search engine plugins capabilities and exits')
     parser.add_argument('--progress', '-p', default=False, action='store_true',
                         help='If set, outputs search progress every now and then')
-    parser.add_argument('--engines-dir', '-d', default=None, dest="engines_dirs", action='append',
+    parser.add_argument('--engines-dir', '-d', default=[path.join(path.dirname(__file__), 'engines')], dest="engines_dirs", action='append',
```

Comment:
There is a check for `None` in `importer.py`, which replaces it with current directory and its parent. I'll roll this back.